### DISCUSSION TITLE
Classic Mac programs

### DISCUSSION THREAD

**Discussion by upintheairsheep** [elrumo/macOS_Big_Sur_icons_replacements#1127] [Icon request]
Can you make icons for legacy MacOS programs?

Here are the specific ones I want on the site.
* iDVD
*  iWeb
* Front Row
* iSync
* iTunes
* iPhoto
* Aperture
* Cinema tools
* DVD Studio Pro

the reason I want to do this is because of Retroactive.

**Comment by ghost** [comment]
[Archiv.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/6242703/Archiv.zip)
Hi, is that what you are searching for or do you mean Big Sur Icons of these programs?

**Reply by upintheairsheep** [reply]
Great! Can you also do https://www.theiphonewiki.com/wiki/Apple_Internal_Apps#macOS_Internal_Apps

**Comment by upintheairsheep** [comment]
Yes, that’s what I want. Big Sur versions of them.

**Reply by ghost** [reply]
my ideas for icons:

iDVD ![IMG_1451](https://user-images.githubusercontent.com/81748804/113326186-a1072e80-9319-11eb-9b7a-e81aaaba9f41.png) 
Aperture ![IMG_1456](https://user-images.githubusercontent.com/81748804/113326201-a5334c00-9319-11eb-91cf-9ad6eef314b0.png)
Front Row ![IMG_1458](https://user-images.githubusercontent.com/81748804/113326224-aebcb400-9319-11eb-8f94-6f7914c82d7b.png)
iPhoto ![IMG_1461](https://user-images.githubusercontent.com/81748804/113326246-b5e3c200-9319-11eb-8ae6-d6eda3a12f4d.png)
iSync ![IMG_1463](https://user-images.githubusercontent.com/81748804/113326267-bb410c80-9319-11eb-9706-31d3999a64ed.png)

**Comment by ghost** [comment]
Okay, I will try to create some for you

**Comment by upintheairsheep** [comment]
They look beautiful. Keep up the good work. But I need the iWeb icon. Also, I request Mac OS Catalina and Mojave downgrade icons (just In case)

**Reply by ghost** [reply]
Yeah, sorry, I tried to make iWeb as well, but didn‘t really managed to bring my idea on the iPad. And thank you very much! (These are just ideas! If you look closely, you‘ll find some mistakes, because these aren‘t final versions)
What do you mean by Catalina and Mojave downgrade icons?

**Comment by upintheairsheep** [comment]
Installer icons.

**Comment by edmechem** [comment]
Hey @upintheairsheep - what's Retroactive?

**Reply by upintheairsheep** [reply]
Tool that patches iTunes, iPhoto, and Aperture to work on modern macOS

**Reply by SoCuul** [reply]
Now that's a slightly late response haha
